cacheback/base.py

```diff
@@ -379,3 +379,44 @@ def process_result(self, result, call, cache_status, sync_fetch=None):
                            was required (ie the result was a cache hit).
         """
         return result
+
+    # --------------------
+    # ASYNC HELPER METHODS
+    # --------------------
+
+    def job_refresh(klass_str, obj_args, obj_kwargs, call_args, call_kwargs):
```

Comment:
You're right. classmethod decorator was missing. I want to keep the functionality in the Job class to keep things together which belong together.